auth/tests/api/helpers.rs

```diff
@@ -108,12 +111,22 @@ impl TestApp {
         self.send_request(request).await
     }
 
-    pub async fn get_jwt_from_api_key(&self, api_key: &str) -> Response {
-        let request = Request::builder()
+    /// If we don't provide a valid admin key, then the`basic_api_key` parameter
+    /// should be of an admin user.
+    pub async fn get_jwt_from_api_key(
+        &self,
+        basic_api_key: &str,
```

Comment:
Should this rather be named something like "user_api_key"?

Reply:
Yes, would be better indeed. Done.